<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>postpop</class>
 <widget class="QDialog" name="postpop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>561</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>691</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255)</string>
    </property>
    <widget class="QLineEdit" name="dl1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>320</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 BT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none; border-radius: 10px; color: grey;   border: 1px solid black;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Until when?</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ps1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>361</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 BT</family>
       <pointsize>12</pointsize>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none; border-radius: 10px; color: grey;   border: 1px solid black;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>What to post?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>276</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 BT</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent; border: none; color: grey;</string>
     </property>
     <property name="text">
      <string>Deadline: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 BT</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent; border: none; color: grey;</string>
     </property>
     <property name="text">
      <string>Post: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="addpost_btn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>400</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Variable Display</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-color: rgb(21, 21, 21); border-radius: 30px; color: white;}

QPushButton:hover {border-radius: 30px; color: white; background-color: rgb(85, 170, 255);}
</string>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QListWidget" name="frm_board">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>481</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-radius: 20px; border: 1px solid grey; 
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tags_pinfo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>69</x>
       <y>41</y>
       <width>130</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 BT</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none; border-radius: 18px; color: black;   border: 1px solid grey;</string>
     </property>
     <property name="text">
      <string>Add Post</string>
     </property>
    </widget>
    <widget class="QPushButton" name="postback_btn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>71</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Variable Display</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {border-radius: 20px; color: white; background-color: rgb(255, 85, 127);}

QPushButton:hover {border-radius: 20px; border: 1px solid #FF557F; color: rgb(255, 85, 127); background-color: white; }
</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QListWidget" name="frm_board_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>421</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-radius: 20px;
background-color: rgb(243, 243, 243);</string>
     </property>
    </widget>
    <zorder>frm_board</zorder>
    <zorder>frm_board_2</zorder>
    <zorder>addpost_btn</zorder>
    <zorder>dl1</zorder>
    <zorder>ps1</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>tags_pinfo</zorder>
    <zorder>postback_btn</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
